src/main/java/edu/uc/inventorymanager/ui/MainController.java

```diff
@@ -5,42 +5,52 @@
 import edu.uc.inventorymanager.dto.ItemStatus;
 import edu.uc.inventorymanager.dto.User;
 import edu.uc.inventorymanager.service.IItemService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.*;
 
 import java.util.List;
 
 
+
 @Controller
 public class MainController {
+
+    static final String USER_NAME = "John Doe";
+    static final String DESCRIPTION = "Des";
+
     @Autowired
     IItemService itemService;
 
+    Logger log = LoggerFactory.getLogger(this.getClass());
+
     /**
      * Handle the / endpoint
      *
      * @return
      */
     @RequestMapping("/")
     public String index(Model model) {
-        Item item = new Item("item", "Des");
+        Item item = new Item("item", DESCRIPTION);
         item.setLocation("locale");
-        item.setAssignee(new User("John Doe"));
+        item.setAssignee(new User(USER_NAME));
         item.setStatus(new ItemStatus("Assigned"));
         model.addAttribute(item);
         return "index";
     }
 
     @RequestMapping("/saveItem")
-    public String saveItem(Item item) throws Exception {
-        itemService.save(item);
+    public String saveItem(Item item) {
+        try {
+            itemService.save(item);
+        }catch (Exception e){
+            log.error("Unable to Save!");
+        }
```

Comment:
Added try catch on saveItem
Instead of throwing exception, it’s better to use a try-catch method on a save function. Especially for a MVC code. For that reason, I added try-catch with the help from our lectures and this link https://spring.io/blog/2013/11/01/exception-handling-in-spring-mvc.

Reply:
Logging is good... but a catch block should really focus on fixing the error.  As this is now, the catch will log the error, and then the program will continue to run, as if nothing wrong happened.  Is that the correct behavior, or is there another path to take when something goes wrong?